src/components/molecules/UiRating/UiRating.vue

```diff
@@ -93,7 +93,7 @@
             <span
               v-bind="textLabelAttrs"
               class="visual-hidden"
-            >{{ defaultProps.translation.stars }}</span>
+            >{{ defaultProps.translation.stars(index) }}</span>
```

Comment:
@ArkadiuszKrzymienski chceck template in storybook for 1 star you have 0, for 2 you have 1 :)

Reply:
Thanks 🙂 I've already fixed it ✌️